Makefile.project

```diff
@@ -0,0 +1,9 @@
+all: verify
+
+build:
+	@cannon build --write-deployments deployments
+
+verify: build
+	@jq -r '"\(.contractName) \(.address)"' deployments/*.json | sort | uniq | diff -u expected -
```

Comment:
This command is long. We could create a target for `@jq [...] | sort | uniq` named `print-addresses`, and make the `verify` target run `@$(MAKE) print-addresses | diff -u expected -`.

```suggestion
print-addresses: build
	@jq -r '"\(.contractName) \(.address)"' deployments/*.json | sort | uniq

verify:
	@$(MAKE) print-addresses | diff -u expected -
```

Reply:
It doesn't work: `No rule to make target `print-addresses'.`
I guess it's because that rule is being call with `-f`?

Reply:
Yeah, never mind. :-(